src/Resources/views/Tree/tree.html.twig

```diff
@@ -24,11 +24,11 @@ file that was distributed with this source code.
 <script type="text/javascript">
     var apiGetTemplate = '{{ path('_cmf_get_resource', {
         repositoryName: repository_name,
-        root_node: root_node,
```

Comment:
I think we still need that node here, isn't it?

Reply:
I don't think so, the route and the controller do not use it : 
https://github.com/symfony-cmf/resource-rest-bundle/blob/039884f80d2864f6457da5988801ae42cad439a9/src/Resources/config/routing.yml#L17
https://github.com/symfony-cmf/resource-rest-bundle/blob/master/src/Controller/ResourceController.php#L68

Reply:
now it makes sense, sorry i was sleeping. Can you remove it in Line 43, too. best would be to use `apiGetTemplate.replace('__path__', sourcePath),` there.

Reply:
Done !